use_cases.md

```diff
@@ -725,6 +765,8 @@ admission webhook "pvc.capsule.clastix.io" denied the request:
 Storage Class default is forbidden for the current Tenant
 ```
 
+If no `storageClasses` is specified in Tenant spec this policy is not enforced and Alice can use any storage class.
```

Comment:
According to change requests, that wouldn't be true.

Reply:
removed